_posts/2015-01-23-drivers.md

```diff
@@ -89,7 +89,7 @@ Drivers should focus solely on being an interface for effects, and usually are l
 
 {% highlight js %}
 run(main, {
-  log: msg$ => { msg$.subscribe(msg => console.log(msg)) }
+  log: msg$ => { msg$.addListener({next: msg => console.log(msg)}) }
```

Comment:
In xstream, if I remember correctly, you must provide all three, `next`, `error` and `complete` properties.